docs/pages/versions/unversioned/sdk/webbrowser.md

```diff
@@ -46,8 +55,7 @@ Returns a Promise:
 
 ### `WebBrowser.openAuthSessionAsync(url, redirectUrl)`
 
-Opens the url with Safari in a modal on iOS using `SFAuthenticationSession`, and Chrome in a new [custom tab](https://developer.chrome.com/multidevice/android/customtabs) on Android. On iOS, the user will be asked whether to allow the app to authenticate using
-the given url.
+Opens the url with Safari in a modal on iOS using `SFAuthenticationSession`. The user will be asked whether to allow the app to authenticate using the given url. Unavailable on Android.
```

Comment:
Was this always unavailable on Android and this is a docs update, or is this PR making this method unavailable on Android? 🤔

Reply:
It was always. There is no special Authentication mode in Custom Tabs.